src/types/UiaTextRangeBase.cpp

```diff
@@ -601,11 +612,15 @@ IFACEMETHODIMP UiaTextRangeBase::Move(_In_ TextUnit unit,
     _outputRowConversions();
 #endif
 
-    auto moveFunc = &_moveByDocument;
+    std::function<std::pair<Endpoint, Endpoint>(gsl::not_null<IUiaData*>, const int, const MoveState, gsl::not_null<int*> const)> moveFunc = &_moveByDocument;
     if (unit == TextUnit::TextUnit_Character)
     {
         moveFunc = &_moveByCharacter;
     }
+    else if (unit <= TextUnit::TextUnit_Word)
+    {
+        moveFunc = std::bind(&_moveByWord, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3, _wordDelimiters, std::placeholders::_4);

```

Comment:
Oh boy. @DHowett-MSFT, there's a way to do this without `std::bind`, isn't there?

Reply:
the expensive thing here is really `std::function`, not so much `std::bind`..
but to save on the cost of a _bind_, we can use a lambda:

```
moveFunc = [=](auto&& arg1, auto&& aeg2, auto&& arg3, auto&& arg4){ return _moveByWord(arg1,arg2,arg3,_wordDelimiters,arg4); }
```

May be cheaper.

Reply:
> `aeg2`

Reply:
aeeeeg